compute_tools/src/bin/fast_import.rs

```diff
@@ -677,11 +678,35 @@ pub(crate) async fn main() -> anyhow::Result<()> {
                 spec,
                 source_connection_string,
                 destination_connection_string,
-                args.working_directory,
+                args.working_directory.clone(),
                 args.pg_bin_dir,
                 args.pg_lib_dir,
             )
-            .await?;
+            .await
+        }
+    };
+
+    if let Some(s3_prefix) = args.s3_prefix {
+        info!("write job status to s3");
+        {
+            let status_dir = args.working_directory.join("status");
+            if std::fs::exists(&status_dir)?.not() {
+                std::fs::create_dir(&status_dir).context("create status directory")?;
+            }
```

Comment:
I was thinking about a separate `workdir` object with some internal folders/file management, maybe indeed makes sense. For now just copied